service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsController.java

```diff
@@ -157,15 +155,19 @@ private void removeDocumentFromData(Map<String, Object> data) {
     private List<Element<C2DocumentBundle>> buildC2DocumentBundle(CaseData caseData) {
```

Comment:
This stuff should be extracted to a service, too much logic for a controller

Reply:
Happy for it to be a future PR though

Reply:
I agree, I already have a ticket for it in support which i created during last sprint.

Reply:
@KiranVarma02 what ticket?

Reply:
https://tools.hmcts.net/jira/browse/FPLA-2186